amundsen_application/static/css/_icons.scss

```diff
@@ -183,6 +183,11 @@ img.icon {
     mask-image: url('/static/images/icons/Up.svg');
   }
 
+  &.icon-user {
+    -webkit-mask-image: url('/static/images/icons/users.svg');
+    mask-image: url('/static/images/icons/users.svg');
```

Comment:
what is the difference between webkit-mask-image vs mask-image ?

Reply:
Different browsers have small differences/complications with how they handle css styles, this is why sometimes UIs run into issues with some things working in one browser and not working in another. 

For the `mask-image` style, Chrome for whatever reason implemented support for this only when prefixed with `-webkit-`, so we need this option to work in Chrome.